scipy/stats/_new_distributions.py

```diff
@@ -200,6 +200,76 @@ def _sample_formula(self, full_shape, rng, **kwargs):
         return rng.normal(size=full_shape)[()]
 
 
+class Logistic(ContinuousDistribution):
+    r"""Standard logistic distribution.
+
+    The probability density function of the standard logistic distribution is:
+
+    .. math::
+
+        f(x) = \frac{1}{\left( e^{x / 2} + e^{-x / 2} \right)^2}
+
+    """
+    _x_support = _RealInterval(endpoints=(-inf, inf))
+    _variable = _x_param = _RealParameter('x', domain=_x_support, typical=(-9, 9))
+    _parameterizations = ()
+
+    _scale = np.pi / np.sqrt(3)
+
+    def _logpdf_formula(self, x, **kwargs):
+        y = -np.abs(x)
+        return y - 2 * special.log1p(np.exp(y))
+
+    def _pdf_formula(self, x, **kwargs):
+        # f(x) = sech(x / 2)**2 / 4
+        return (.5 / np.cosh(x / 2))**2
+
+    def _logcdf_formula(self, x, **kwargs):
+        return special.log_expit(x)
+
+    def _cdf_formula(self, x, **kwargs):
+        return special.expit(x)
+
+    def _logccdf_formula(self, x, **kwargs):
+        return special.log_expit(-x)
+
+    def _ccdf_formula(self, x, **kwargs):
+        return special.expit(-x)
+
+    def _icdf_formula(self, x, **kwargs):
+        return special.logit(x)
+
+    def _iccdf_formula(self, x, **kwargs):
+        return -special.logit(x)
+
+    def _entropy_formula(self, **kwargs):
+        return 2.0
+
+    def _logentropy_formula(self, **kwargs):
+        return np.log(2)
```

Comment:
Nit, optional: math.log(2) ? 
And similar for the _scale classvar

Reply:
I was first thinking of going with `scipy.integrate.cubature(lambda t: 1/t, [1], [2]).estimate.item()` but I like your idea better :P

Reply:
Hmm, it'll be a inconsistent with `Normal` this way 🤔 

https://github.com/scipy/scipy/blob/d206e414575046a315781b1a8cfc9a167fd8d86f/scipy/stats/_new_distributions.py#L41-L42

It also requires an additional `math` import.

So do we still want this?

Reply:
Go ahead and change `Normal` to use `math`, too.

Reply:
I think that it would be cleaner to address that in a separate PR.